doodle/docs/source/intro/overview.rst

```diff
@@ -0,0 +1,15 @@
+.. _intro-overview:
+
+==========================
+Pytheranostics at a glance
+==========================
+
+PyTheranostics is a library of tools to process nuclear medicine scans and take them through the dosimetry workflow to calculate the absorbed dose in target organs. The library is designed to be modular and extensible, so that new features can be added easily.
```

Comment:
I would add "... process nuclear medicine scans **with organ and lesion segmentation masks** and take them through ..."